db/primary_migrate/20250115145923_add_ui_locale_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddUiLocaleToUser < ActiveRecord::Migration[7.2]
+  def change
+    add_column :users, :locale, :string, comment: "sensitive=false"
```

Comment:
I would consider reusing (or maybe via `alias_attribute`) the existing `email_language` column.

or alternatively we should document why this one is different, such as this one is set implicitly via user accept language headers (or the UI dropfown) vs the email one which is set explicitly via settings

Reply:
Yeah, when it was named `ui_locale` I almost liked that more since it was kind of a self-documenting way to differentiate it from `email_locale`. Wdyt of something like changing the attribute name to something like `web_locale` to achieve the same? We could keep the scope named `locale`.

Reply:
Alternatively, could we store it in the session?

Reply:
`web_locale` is pretty clear but I think that might come with the expectation it is always updated, so I'd definitely augment the comment with more that it's only updated infrequently

Reply:
Would it be worth aligning "language" vs. "locale" (see existing `email_language`)

Reply:
> Alternatively, could we store it in the session?

I opened #11756 to see what this would look like. It is a similar implementation with testing be a little finickier since we need to go through `OutOfBandSessionAccessor` to get to the locale

Reply:
> Would it be worth aligning "language" vs. "locale" (see existing email_language)

I'll put up a commit to make that change shortly